API.yaml

```diff
@@ -489,6 +489,92 @@ paths:
             application/json:
               schema:
                 type: string
+ /experiments/{expUUID}/annotations:
+    get:
+      summary: Get the collection of annotations categories for an experiment.
+      operationId: getExperimentAnnotations
+      tags:
+      - Annotations
+      parameters:
+      - name: expUUID
+        in: path
+        description: The UUID of the experiment in the server
+        required: true
+        schema:
+          type : integer
+          format: int128
+      requestBody:
+        description: Query parameters to fetch the annotations.
+          Usually contains these parameters, but not limited to
+          - list of outputIds
+          - startTime
+          - endTime
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Query'
+      responses:
+        200:
+          description: Returns a list of annotation categories for this trace
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Bookmark'
+        404:
+          description: No such experiment
+          content:
+            application/json:
+              schema:
+                type: string
+/experiments/{expUUID}/annotations/{annotationId}:
+    get:
+      summary: Get a list of specific annotations from this experiment
+      operationId: getExperimentAnnotation
+      tags:
+      - Annotations
+      parameters:
+      - name: expUUID
+        in: path
+        description: The UUID of the experiment in the server
+        required: true
+        schema:
+          type : integer
+          format: int128
+      requestBody:
+        description: Query parameters to fetch the annotations.
+          Usually contains these parameters, but not limited to
+          - list of outputIds
+          - startTime
+          - endTime
+          - categoryID
+          - annotationId
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Query'
+      responses:
+        200:
+          description: Returns the queried annotations (an array of arrays of arrays , category -> outputID -> annotations)
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Annotations'
```

Comment:
This is not defined, Annotations should be generic and use _allOf_ to add specific fields for XY and TimeGraph